serverless/pages/ml-nlp-auto-scale.mdx

```diff
@@ -0,0 +1,128 @@
+---
+slug: /serverless/general/ml-nlp-auto-scale
+title: Trained model autoscaling
+tags: ['serverless']
+---
+
+You can enable autoscaling for each of your trained model deployments.
+Autoscaling allows Elasticsearch to automatically adjust the resources the model deployment can use based on the workload demand.
+
+There are two ways to enable autoscaling:
+
+- through APIs by enabling adaptive allocations
+- in Kibana by enabling adaptive resources
+
+
+Trained model autoscaling is available for Search, Observability, and Security projects on serverless deployments. However, these projects handle processing power differently, which impacts their costs and resource limits.
+
+Security and Observability projects are only charged for data ingestion and retention. They are not charged for processing power (vCU usage), which is used for more complex operations, like running advanced search models. For example, in Search projects, models such as ELSER require significant processing power to provide more accurate search results.
+
+## Enabling autoscaling through APIs - adaptive allocations
+
+Model allocations are independent units of work for NLP tasks.
+If you set the numbers of threads and allocations for a model manually, they remain constant even when not all the available resources are fully used or when the load on the model requires more resources.
```

Comment:
Not sure if `manually` is the right word here. I'd say `if you set a static number of allocations`.